### PR TITLE
Migrate CLX DGA Detection

### PR DESCRIPTION
**Pull request by efajardo-nv**
## Description
- Migrate code, notebook, dataset for CLX DGA Detection
- Remove references to CLX
- Example Morpheus pipeline
- Triton model repo with ONNX model
- Fix bug in existing code caused by upstream cudf updates
- Add new training and validation data
- Merge version updates from PR #49
- Remove `cuml` dependency so scripts/notebooks can also be run in Morpheus container 

Task from https://github.com/nv-morpheus/Morpheus/issues/508

## Checklist
[x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
[x] New or existing tests cover these changes.
[x] The documentation is up to date with these changes.

### CODE REVIEW

**Review by mdemoret-nv — APPROVED** [review]

File: dga-detection/morpheus-pipeline/preprocessing.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 import cupy as cp
 import mrc
-from messages import InferenceMemoryDGA
+# from messages import InferenceMemoryDGA
```
Comment (mdemoret-nv) [inline dga-detection/morpheus-pipeline/preprocessing.py:20]: Left over commented code

**Review by mdemoret-nv — APPROVED** [review]
Merge once the comment has been fixed

**Review by raykallen — APPROVED** [review]

**Pull request merged**